src/components/detailPage.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DetailPage = ({ image }) => (
+  <div>
+    <img src={image} alt="Detail" />
+    {/* Other component content will be rendered here */}
+  </div>
+);
```

Comment:
Kindly, render Unit details here according to [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) provided by Microverse and API unit show endpoint.

### Ticket details here:
    Create a page to display item/unit details and photo and allow users to book a visit.

    Show page
        When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description (skip the "Rotate image" button).
        In the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).
    Styling and responsiveness
        Make the app responsive, creating both mobile and desktop versions.
        You should follow the layout of[ the provided design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign). You should change only the titles, descriptions, and photos - in order to create a website about something other than motorcycles.